buildbot

```diff
@@ -12,9 +12,13 @@ def echo_run(cmd):
     print('+', *cmd)
     run(cmd, check=True)
 
+def get_vm_name():
+    process = run(['kitchen', 'list', '--bare'], stdout=PIPE)
```

Comment:
> Maybe we want to check the return code of this command?

Yep, pushing a fix now.

> Also, how does this work if kitchen list lists two vms?

nowadays, `.kitchen.yml` is [generated in `prepare_cloud_image.py`](https://github.com/liquidinvestigations/buildbot/blob/d556bc7308674971305b6b617db4735f38c9b161/prepare_cloud_image.py#L36), and it contains just one vm.